composer.json

```diff
@@ -25,7 +25,8 @@
 		"php": "^7.1 || ^8.0",
 		"webonyx/graphql-php": "14.11.10",
 		"ivome/graphql-relay-php": "0.7.0",
-		"appsero/client": "1.2.1"
+		"appsero/client": "1.2.1",
+		"justinrainbow/json-schema": "^5.2"
```

Comment:
If we're validating in a GH action, do we really need an entire dependency to validate inside the plugin too?

PS: storing as a signed PHP array would allow us to skip _both_ the GH workflow _and_ the dependency.